ScanbotSDKExamples/NET/ReadyToUseUI.Droid/MainActivity.cs

```diff
@@ -178,6 +178,19 @@ private void OnButtonClick(object sender, EventArgs e)
                 var intent = DocumentScannerActivity.NewIntent(this, configuration);
                 StartActivityForResult(intent, Constants.CAMERA_DEFAULT_UI_REQUEST_CODE);
             }
+            else if (button.Data.Code == ListItemCode.ScanDocumentWithFinder)
+            {
+                var configuration = new FinderDocumentScannerConfiguration();
+                configuration.SetCameraPreviewMode(CameraPreviewMode.FitIn);
+                configuration.SetIgnoreBadAspectRatio(true);
+                configuration.SetTextHintOK("Don't move.\nCapturing document...");
+                //configuration.SetFinderLineColor(Color.Blue);
```

Comment:
we can remove these comments

Reply:
These are left as an example, but I have moved the configuration lines under the comment.